lib/dry/component/container.rb

```diff
@@ -123,6 +123,8 @@ def self.require(*paths)
       end
 
       def self.load_component(key)
+        return if key?(key)
```

Comment:
Oh, right :) We don't operate on `.load_component`'s return value anywhere, so I think a `nil` return in this case would be fine.

Perhaps I can make this PR more robust by adding a small yardoc snippet to the top making it clear this is a method with a "void" return?